### ISSUE TITLE
Lando tooling: Multi row command parsing

### ISSUE THREAD

**Issue by martin-klima** [lando/lando#703]
Bug Report
==========

**Tell us about your setup**

*What is your lando version and operating system?*
Linux ProBook 4.10.0-38-generic #42~16.04.1-Ubuntu SMP Tue Oct 10 16:32:20 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux + Lando v3.0.0-beta.35

**Tell us about your `.lando.yml`**

```
name: test
recipe: drupal7

services:
  appserver:
    type: php:7.0
    webroot: docroot
    via: apache
    ssl: true
    xdebug: true

tooling:
  test:
    service: appserver
    cmd:
      - cd /
      - ls
      - ls *.*

```

**Tell us about the command you were running**

```bash
lando test -- -vvv
```

**Tell us about the error you got**

```bash
verbose: Exec cd "$LANDO_MOUNT"&&'cd /' ls 'ls *.*' on test_appserver_1 in mode attach
```

**Tell us generally about your bug**

1) The commands with spaces are automatically enclosed to `'`. It is problem for `cd` command.
Command `cd "$LANDO_MOUNT"&&'cd /'` causes error `/bin/sh: 1: cd /: not found`.
Command `cd "$LANDO_MOUNT"&&cd /` without quotes works well and changes the dir. 

2) Only first and second command are separated with `&&` and all the other commands are separated by space. It causes that from the 3rd command none of them are called.
Parsed command `cd "$LANDO_MOUNT"&&'cd /' ls 'ls *.*'` should be parsed as
`cd "$LANDO_MOUNT" && cd / && ls && ls *.*` (I added a spaces for better readability)

More generally
`cd "$LANDO_MOUNT"&&'command 1' 'command 2' 'command 3' 'command 4'`
should be
`cd "$LANDO_MOUNT" && command 1 && command 2 && command 3 && command 4`

Thank you for all Lando features.

**Comment by dustinleblanc** [comment]
@martin-klima Do you get the same outcome if you fully quote the command on each line (`'cd /'`)?

**Comment by martin-klima** [comment]
Yes. For quoted commands in .lando.yml:
```
tooling:
  test:
    service: appserver
    cmd:
      - 'cd /'
      - 'ls'
      - 'ls *.*' 
```
I get:
```
verbose: Exec cd "$LANDO_MOUNT"&&'cd /' ls 'ls *.*' on test_appserver_1 in mode attach

debug: With opts {"AttachStdin":true,"AttachStdout":true,"AttachStderr":true,"Cmd":["/bin/sh","-c","cd \"$LANDO_MOUNT\"&&'cd /' ls 'ls *.*'"],"Env":[],"DetachKeys":"ctrl-p,ctrl-q","Tty":true,"User":"www-data"} {"hijack":false,"stdin":true,"Detach":false,"Tty":true}

/bin/sh: 1: cd /: not found
```

**Comment by OriginalEXE** [comment]
Hello, and thank you for maintaining this useful project!

I was wondering if there was any chance this could get prioritized? This bug is really annoying and I had to instead write my tooling commands as a shell script which I run with my tooling.

**Comment by dustinleblanc** [comment]
Hey @martin-klima and @OriginalEXE we just talked about this issue in our maintainers meeting and we realized what is going on here. B

In the context of tooling, Lando just space separates  each key in the array. This works for splitting up commands like `drush arg1 arg2 --option` which is how I've always used the tooling entries, hence why I never see a problem with them.

The way this kind of 'array of stuff' works in pretty much everything else (like build steps) is that it runs a new command for each line. This is generating confusion as you would rightfully expect that API to be consistent and it isn't right now.

We're thinking about how to address this, as consistency makes sense here, but if we just willy nilly change it, we could break the experience for a lot of folks as well as some of our example docs.

There are a couple of work arounds here, like just running the command as a big string, to putting the steps in a shell script (or grunt task, or robo script, etc) and just run them with some like `cmd: ./script.sh` or `cmd: robo command name` (which is how I always use tooling).

For now we're going to defer this until we've had a chance to think it over.
